prac_05/hex_colours.py

```diff
@@ -0,0 +1,15 @@
+COLOUR_TO_HEX = {"absolute zero": "#0048ba", "acid green": "#b0bf1a", "aliceblue": "#f0f8ff",
+               "alizarin crimson": "#e32636", "amaranth": "#e52b50", "amber": "#ffbf00", "amethyst": "#9966cc",
+               "antiquewhite": "#faebd7", "antiquewhite1": "#ffefdb", "antiquewhite2": "#eedfcc",
+               "antiquewhite3": "#cdc0b0", "antiquewhite4": "#8b8378", "apricot": "#fbceb1", "aqua": "#00ffff"}
+
+
+while True:
+    colour_input = input("Enter Colour Name: ").lower()
+    if (colour_input == ""):
+        print("End.")
+        break
```

Comment:
Would it be better to have a conditional while loop that ends when colour_input is empty rather than having this extra if statement?